src/features/synth/utils.ts

```diff
@@ -36,6 +36,15 @@ function getAudioContext() {
   return AudioContext
 }
 
+// Node that listens all gainNotes for recording purposes
+let recordingDestinationNode: MediaStreamAudioDestinationNode
```

Comment:
Can you instantiate `recordingDestinationNode` within `getRecordDestinationNode` (singleton pattern).

Similar to this:
```js
// Returns the DestinationNode that all user-played sounds are routed through
function getRecordingDestinationNode() {
  if (!recordingDestinationNode && audioContext != null) {
    recordingDestinationNode = ...
  }
  return recordingDestinationNode;
}
```

Reply:
Ok